UnitsNet/CustomCode/Quantities/Length.extra.cs

```diff
@@ -80,7 +101,8 @@ public static bool TryParseFeetInches(string? str, out Length result, IFormatPro
             str = str.Trim();
 
             // This succeeds if only feet or inches are given, not both
-            if (TryParse(str, formatProvider, out result))
+            // Do not allow thousands separator here, since it may be equal to the unit abbreviation (').
+            if (TryParse(str, formatProvider, NumberStyles.Float, out result))
```

Comment:
As @tmilnthorp pointed out earlier, we should have unit tests that covers the new behavior for feet-inches. It should test what happens if you try to parse strings like these, with a culture that has apostrophe character as thousand separator. We could also test cultures with other thousand separators, like `nb-NO` Norwegian that has comma `,` as separator. 
```
1'500' 4"
1'5002′−4″
1'500 ft 4 in
1'5002′ 4″
``` 

My intuition tells me we should get an exception when parsing these, because the thousand separator is no longer allowed and in the first case the feet symbol is even listed twice.

Reply:
I'll do that, but I feel we should somehow merge @tmilnthorp s PR with this one, otherwise it gets confusing.

Reply:
I guess we can wait for #799 to merge first to reuse the combinatorial parameter refactoring that is about to be added there.

Reply:
#799 was abandonend, please see #803 for supporting thousand separators in regex instead of not allowing them. I think that may be a better solution to this problem.